tools/BUILD.md

```diff
@@ -0,0 +1,72 @@
+Build Instructions
+==================
+
+Quickstart
+----------
+
+### Steps
+
+*Ensure there are no ROS environment variables set in your terminal or `.bashrc` file before taking the steps below.*
+
+```bash
+mkdir <workspace_dir>
+cd <workspace_dir>
+wget https://raw.githubusercontent.com/ros-planning/navigation2/buildscript/tools/initial_ros_setup.bash
```

Comment:
Should file names be changed from .bash to .sh since you changed the shell to sh?

Reply:
Yes. Done with the latest checkin.